src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/CaseTypeTab.java

```diff
@@ -475,6 +507,24 @@ private void buildOutcomeOfConditionalOrderTab(ConfigBuilder<CaseData, State, Us
             .field("coRescindedDate");
     }
 
+    private void buildOutcomeOfConditionalOrderTabApp2Sol(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        final Tab.TabBuilder<CaseData, UserRole> tabBuilder = configBuilder.tab(
+                "outcomeOfConditionalOrderApp2Sol", "Outcome of Conditional Order")
+            .forRoles(APPLICANT_2_SOLICITOR)
+            .showCondition("applicationType=\"jointApplication\" AND coGranted=\"*\" OR "
+                    + showForState(
+                    AwaitingAdminClarification,
+                    AwaitingClarification,
+                    AwaitingAmendedApplication,
+                    AwaitingPronouncement,
+                    ClarificationSubmitted,
+                    LAReview,
+                    SeparationOrderGranted
+                )
+            );
+        addOutcomeOfConditionalOrderTabFields(tabBuilder);
+    }
```

Comment:
Nitpick: If we move this above `addOutcomeOfConditionalOrderTabFields` it will be sat next to the equivalent method for applicant 1 solicitor and easier for someone to compare the two versions of the tab.

I also wonder whether we could pass the first part of the showCondition (`"applicationType=\"jointApplication\""`) into addOutcomeOfConditionalOrderTabFields() and put the duplicated show condition code in the shared method? e.g:

```
private void addOutcomeOfConditionalOrderTabFields(final Tab.TabBuilder<CaseData, UserRole> tabBuilder, String showCondition) {
  tabBuilder.showCondition(showCondition + " AND coGranted=\"*\" OR "
                    + showForState(
                    AwaitingAdminClarification,
                    AwaitingClarification,
                    AwaitingAmendedApplication,
                    AwaitingPronouncement,
                    ClarificationSubmitted,
                    LAReview,
                    SeparationOrderGranted
                )
            )
}
```